core/templates/dev/head/domain/exploration/ParamSpecObjectFactorySpec.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ParamSpecObjectFactory.
+ */
+import { TestBed } from '@angular/core/testing';
+
+import { ParamSpecObjectFactory } from
+  'domain/exploration/ParamSpecObjectFactory';
+import { ParamTypeObjectFactory } from
+  'domain/exploration/ParamTypeObjectFactory';
+
+describe('Param Spec Object Factory', () => {
+  let psof, ptof;
+  let paramType;
+
+  beforeEach(() => {
+    psof = TestBed.get(ParamSpecObjectFactory);
+    ptof = TestBed.get(ParamTypeObjectFactory);
+
+    paramType = ptof.getDefaultType();
+  });
+
+  it('should create a param spec object from backend', () => {
```

Comment:
```suggestion
  it('should create a param spec object from backend dict', () => {
```

Please change this elsewhere (in other PRs too).

Reply:
Done.